src/JuegoTriangulos.java

```diff
@@ -1,8 +1,138 @@
+
+        // Crear un Scanner para leer la entrada
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Ingrese una opción: ");
+
+        int opcion = -1;
+        while (opcion != 5) {
+            if (scanner.hasNextInt()) {
+                opcion = scanner.nextInt();
+                scanner.nextLine(); // Consumir el salto de línea
+                switch (opcion) {
+                    case 1:
+                        registrarJugador(scanner);
+                        // Aquí puedes agregar la lógica para registrar jugadores
+                        break;
+                    case 2:
+                        System.out.println("Opción 2 seleccionada: Configurar partida");
+                        // Aquí puedes agregar la lógica para configurar la partida
+                        break;
+                    case 3:
+                        System.out.println("Opción 3 seleccionada: Jugar partida");
+                        // Aquí puedes agregar la lógica para jugar la partida
+                        break;
+                    case 4:
+                        System.out.println("Opción 4 seleccionada: Mostrar ranking");
+                        // Aquí puedes agregar la lógica para mostrar el ranking
+                        break;
+                    case 5:
+                        System.out.println("Saliendo del juego. ¡Hasta luego!");
+                        break;
+                    default:
+                        System.out.println("Opción inválida. Por favor, ingrese un número entre 1 y 5.");
+                }
+            } else {
+                System.out.println("Entrada inválida. Por favor, ingrese un número.");
+                scanner.next(); // Consumir la entrada inválida
+            }
+            if (opcion != 5) {
+                System.out.println("");
+                mostrarMenu();
+                System.out.println("\nIngrese una opción: ");
+            }
+        }
+        scanner.close();
+
     }
-} 
+
+    public static void mostrarMenu() {
+        System.out.println(" --- MENÚ PRINCIPAL ---");
+        System.out.println("1. Registrar jugador");
+        System.out.println("2. Configurar partida");
+        System.out.println("3. Jugar partida");
+        System.out.println("4. Mostrar ranking");
+        System.out.println("5. Salir");
+        System.out.println("----------------------");
+    }
+
+    public static void registrarJugador(Scanner scanner) {
+        if (jugadoresRegistrados >= MAX_JUGADORES) {
+            System.out.println("");
+            System.out.println("*-------------------------------------------*");
+            System.out.println("No se pueden registrar más de dos jugadores.");
+            System.out.println("*-------------------------------------------*");
+            return;
+        }
+        System.out.println("");
+
+        // Pedir el nombre del jugador y verificar unicidad
+        String nombre;
+        boolean nombreUnico;
```

Comment:
Las variables que son booleanas tiene que nombrarse como si fueran una pregunta que el valor de la variable responde. Ej: el nombre es unico? si o no, se entiende? Es solo una maña de los programadores, cambialo si queres, sino tampoco me genera mucho drama, porque se entiende clarito, pero para que lo tengas ahi en mente.
```suggestion
        boolean esNombreUnico;
```